lib/drivers/dexcomDriver.js

```diff
@@ -631,6 +628,9 @@ module.exports = function (config) {
           threshold: 400
         };
       }
+      if (!common.isValidLocalTimestamp(datum.displayTime, datum.displayUtc, sundial.getOffsetFromZone(datum.displayUtc, cfg.timezone))) {
+        datum.annotation = {code: 'spring-forward'};
```

Comment:
Only the data that has the invalid local timestamp will be annotated (that will also help us identify it as opposed to the other half of the overlap). Because of the Dexcom sort order, the data points in the non-invalid-timestamp half of the data are actually the ones being edited :/ I thought about introducing an annotation there to reflect the fact that we're adding a millisecond to the timestamp, but then I thought that info is already recoverable because the `deviceTime` is there without the millisecond addition. Open to changing this if others feel strongly about having an annotation whenever we add the millisecond (I think it should be a different one though, so that `spring-forward` only identifies data that originally had timestamps that are invalid in the local timezone).